Tests/GRPCCodeGenTests/Test_TextBasedRenderer.swift

```diff
@@ -0,0 +1,781 @@
+/*
```

Comment:
re: filename, I know you copy-pasted from OpenAPI, but naming is generally `<Foo>Tests.swift`, this should be `TextBasedRendererTests.swift`. We're also trying to keep the hierarchy of library and test code the same so it should be `GRPCCodeGenTests/Internal/Renderer/TextBasedRendererTests.swift`.